### PR TITLE
Update dependencies and add package-lock.json.

### LINKED ISSUE #265: Update dependencies
@bezoerb: if you release a new version, can you also update the deps? There's a security warning for debug.

### PR DESCRIPTION
**Pull request by XhmikosR**
Fixes #265.

Note that this is untested since there are some failures we were getting before this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "indent-string": "^3.1.0",
     "inline-critical": "^3.0.0",
     "lodash": "^4.17.4",
-    "meow": "^3.7.0",
+    "meow": "^4.0.0",
```
Comment (sindresorhus) [inline package.json]: Meow v4 has breaking changes. You need to adapt to those: https://github.com/sindresorhus/meow/releases/tag/v4.0.0 (Note how specifying `minimist` options changed)

Comment (sindresorhus) [reply package.json]: This needs to be changed: https://github.com/addyosmani/critical/blob/2f332dc8986e31b820b33f2e2cd98b4816b3d81a/cli.js#L43-L57

Comment (XhmikosR) [reply package.json]: Thanks. I haven't even got so far, because tests fail anyway...

**Comment by XhmikosR** [comment]
Actually, I'm gonna close this for now.

**Pull request closed without merging**